### ISSUE TITLE
`method` keyword in `pandas.dataframe.replace` now deprecated and will be removed in future pandas version

### ISSUE THREAD

**Issue by ld-archer** [Leeds-MRG/Minos#383]
Recieved following warning when running through US_missing_main:
```
/home/luke/Documents/WORK/MINOS/Minos/minos/data_generation/US_missing_LOCF.py:69: FutureWarning: The 'method' keyword in Series.replace is deprecated and will be removed in a future version.
  return pid_groupby.apply(lambda x: x.replace(US_utils.missing_types, method="ffill").replace(US_utils.missing_types, method="bfill"))
```

The `method` keyword argument in `pandas.dataframe.replace()` is being removed, so we need to replace this functionality. The best way to do this is to use the `dataframe.fillna()` function with the `method` argument (ffill or bfill):

```
apply(lambda x: x.fillna(method="ffill"))
```

**Comment by RobertClay** [comment]
I think its been split up into several functions https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.ffill.html.

hopefully not hard to just replace this with ffill bfill functions

**Comment by ld-archer** [comment]
https://stackoverflow.com/questions/65427188/pandas-fillnamethod-ffill-vs-ffill

fillna() quicker.

**Comment by RobertClay** [comment]
even better.

**Comment by ld-archer** [comment]
Fixed in PR #437
